Tests/Unit Tests/Conditions/PostcodeConditionTests.swift

```diff
@@ -16,48 +16,187 @@ final class PostcodeConditionTests: XCTestCase {
     
     // MARK: - Properties
     
-    let condition       = PostcodeCondition(country: .UnitedKingdom)
+    let conditionGBR        = PostcodeCondition(country: .UnitedKingdom)
+    let conditionSWE        = PostcodeCondition(country: .Sweden)
+    let conditionTUR        = PostcodeCondition(country: .Turkey)
     
     
-    // MARK: - Test Success
+    // MARK: - GBR - Test Success
```

Comment:
Good thinking. I applied the same set of tests to just different conditions there, so there's plenty of space to for refactoring. 👍